Project MUSE.js

```diff
@@ -188,75 +194,144 @@ var testCases = [
 				"issue": "4",
-				"language": "en",
 				"libraryCatalog": "Project MUSE",
 				"pages": "735-745",
 				"publicationTitle": "Technology and Culture",
 				"shortTitle": "The Pill at Fifty",
 				"url": "https://muse.jhu.edu/article/530509",
 				"volume": "54",
-				"attachments": [
+				"attachments": [],
+				"tags": [],
+				"notes": [
 					{
-						"title": "Full Text PDF",
-						"mimeType": "application/pdf"
+						"note": "<p>Volume 54, Number 4, October 2013</p>"
 					}
 				],
-				"tags": [],
-				"notes": [],
 				"seeAlso": []
 			}
 		]
 	},
 	{
 		"type": "web",
-		"url": "http://muse.jhu.edu/article/551992",
+		"url": "https://muse.jhu.edu/article/551992",
 		"items": [
 			{
 				"itemType": "journalArticle",
-				"title": "Accountability and Corruption in Argentina During the Kirchners' Era",
+				"title": "Accountability and Corruption in Argentina During the Kirchners’ Era",
 				"creators": [
 					{
-						"firstName": "Luigi",
 						"lastName": "Manzetti",
+						"firstName": "Luigi",
 						"creatorType": "author"
 					}
 				],
-				"date": "2014-08-13",
+				"date": "2014",
 				"DOI": "10.1353/lar.2014.0030",
 				"ISSN": "1542-4278",
 				"abstractNote": "This article highlights an important paradox: in Argentina between 2003 and 2013 the center-left Peronist government’s approach to governance mirrors that of the center-right Peronist administration of the 1990s. While the latter centralized authority to pursue neoliberal reforms, the former have centralized authority in the name of expanding government intervention in the economy. In both cases, corruption has tended to go unchecked due to insufficient government accountability. Therefore, although economic policies and political rhetoric have changed dramatically, government corruption remains a constant of the Argentine political system due to the executive branch’s ability to emasculate constitutional checks and balances.",
 				"issue": "2",
-				"language": "en",
 				"libraryCatalog": "Project MUSE",
 				"pages": "173-195",
 				"publicationTitle": "Latin American Research Review",
 				"url": "https://muse.jhu.edu/article/551992",
 				"volume": "49",
-				"attachments": [
+				"attachments": [],
+				"tags": [],
+				"notes": [
 					{
-						"title": "Full Text PDF",
-						"mimeType": "application/pdf"
+						"note": "<p>Volume 49, Number 2, 2014</p>"
```

Comment:
Discard the notes -- they don't contain additional information

Reply:
thanks for the code review. yes, i discard it.